org.eclipse.xtext.xtext.wizard/src/org/eclipse/xtext/xtext/wizard/Scope.java

```diff
@@ -8,9 +8,9 @@
 package org.eclipse.xtext.xtext.wizard;
 
 public enum Scope {
-	COMPILE("compile", "compile"),
-	TESTCOMPILE("test", "testCompile"),
-	TESTRUNTIME("test", "testRuntime"),
+	COMPILE("compile", "implementation"),
```

Comment:
I'd probably split this into `API` and `IMPLEMENTATION`. You can map them both to `compile` in Maven and to `api`/`implementation` in Gradle.

Reply:
I don’t think we will have implementation ever then as we don’t distinguish between the two in eclipse and thus won’t in Gradle

Reply:
I see, you mean that we reexport everything? In that case `api` would be a more appropriate mapping.

Reply:
Yes and no. We at least export at least one package and thus the likelihood that that package contains something that leaks into api is quite high

Reply:
but in the mydsl case we replicate the dependency to mydsl to mydsl.ide, mydsl.web and mydsl.test and thus cirumvent the problem at all. this is why implementation works and i used it here

Reply:
Yeah, but that's not really a nice pattern. It's clearly an API dependency and shouldn't have to be repeated all the time :)

Reply:
Yes but I don’t know how to get in under one hood with Gradle and Maven.
And thus would repeat the dependency even with api

Reply:
Same with p2 we don’t do reexport and thus need to duplicate deps 
https://github.com/eclipse/xtext-core/blob/0231bea4e46427ae7175adb70cc5888b7d7bd751/org.eclipse.xtext.xtext.wizard/src/org/eclipse/xtext/xtext/wizard/ProjectDescriptor.xtend#L139